account_operating_unit_sequence/models/account_payment.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+MAP_SEQUENCE = {
```

Comment:
Transfer can be here too with payment type = false?

Reply:
Could I resolve by add condition before use `MAP_SEQUENCE`?

Example:
```
if vals["partner_type"] and vals["payment_type"]:
    sequence_code = ou[MAP_SEQUENCE[vals["partner_type"]][vals["payment_type"]]].code
```